src/test/java/org/opentripplanner/routing/algorithm/mapping/__snapshots__/TransitSnapshotTest.snap

```diff
@@ -6314,10 +6314,62 @@ org.opentripplanner.routing.algorithm.mapping.TransitSnapshotTest.test_trip_plan
               "absoluteDirection": "WEST",
               "area": false,
               "bogusName": false,
-              "distance": 1470.24,
+              "distance": 5.54,
```

Comment:
I tried hard to figure out why the results are different in this case but I failed. The new path takes a shortcut through a park which seems like an improvement to me.